Updates to Steps.tex

shujaatkhan opened the pull request:
Includes updates to Steps.tex based on Maggie’s suggestions.
Note: The other files associated with this tex file (pdf, html, etc.)
have not been updated.

llorracc-git commented:
Thanks; edits look good; have pulled them in, and added the .pdf and other
LaTeX-generated files to .gitignore


On Fri, Sep 8, 2017 at 2:59 PM, shujaatkhan <notifications@github.com>
wrote:

> Includes updates to Steps.tex based on Maggie’s suggestions.
> Note: The other files associated with this tex file (pdf, html, etc.)
> have not been updated.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/llorracc/JobMarket/pull/6
> Commit Summary
>
>    - Updates to Steps.tex
>
> File Changes
>
>    - *M* Steps/Steps.tex
>    <https://github.com/llorracc/JobMarket/pull/6/files#diff-0> (108)
>
> Patch Links:
>
>    - https://github.com/llorracc/JobMarket/pull/6.patch
>    - https://github.com/llorracc/JobMarket/pull/6.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/llorracc/JobMarket/pull/6>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGRCJAQpwjaU1Lc2LPqZoQm8mKnUms3Yks5sgY58gaJpZM4PRkSR>
> .
>



-- 
- Chris Carroll

shujaatkhan commented:
I’ve fixed/updated all the link in the Steps file; however, I haven’t been able to find a replacement for the following link:
http://econ.jhu.edu/jobmarket/information/JobMarketComputerHelp.html

I will check with Nina and issue another pull request after I hear back from her.

Shujaat

From: llorracc-git <notifications@github.com>
Reply-To: llorracc/JobMarket <reply@reply.github.com>
Date: Friday, September 8, 2017 at 4:12 PM
To: llorracc/JobMarket <JobMarket@noreply.github.com>
Cc: Shujaat Khan <sakhan@jhu.edu>, Author <author@noreply.github.com>
Subject: Re: [llorracc/JobMarket] Updates to Steps.tex (#6)

Thanks; edits look good; have pulled them in, and added the .pdf and other
LaTeX-generated files to .gitignore


On Fri, Sep 8, 2017 at 2:59 PM, shujaatkhan <notifications@github.com>
wrote:

> Includes updates to Steps.tex based on Maggie’s suggestions.
> Note: The other files associated with this tex file (pdf, html, etc.)
> have not been updated.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/llorracc/JobMarket/pull/6
> Commit Summary
>
> - Updates to Steps.tex
>
> File Changes
>
> - *M* Steps/Steps.tex
> <https://github.com/llorracc/JobMarket/pull/6/files#diff-0> (108)
>
> Patch Links:
>
> - https://github.com/llorracc/JobMarket/pull/6.patch
> - https://github.com/llorracc/JobMarket/pull/6.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/llorracc/JobMarket/pull/6>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGRCJAQpwjaU1Lc2LPqZoQm8mKnUms3Yks5sgY58gaJpZM4PRkSR>
> .
>



--
- Chris Carroll


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/llorracc/JobMarket/pull/6#issuecomment-328202581>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATHpCfsqCaahZa_6cxJlxMa_yrmTIGIDks5sgZ--gaJpZM4PRkSR>.
